spec/presenters/guide_presenter_spec.rb

```diff
@@ -41,6 +41,11 @@
       )
     end
 
+    it "doesn't contain public_updated_at so that it represents the published at time" \
```

Comment:
My brain struggles with this sentence. Might be the negation combined with the 'so that' clause? Not sure what would be better, but maybe changing 'doesn't contain' to e.g. 'omits' might start to clear things up?